packages/sdk/ts/key/key.ts

```diff
@@ -1,7 +1,7 @@
 import { VkRegistry__factory as VkRegistryFactory, extractVk } from "maci-contracts";
 import { IVkContractParams, VerifyingKey } from "maci-domainobjs";
 
-import type { GetAllVksArgs, IExtractAllVksArgs, IMaciVks, IMaciVerifyingKeys } from "./utils/types";
+import type { IGetAllVksArgs, IExtractAllVksArgs, IMaciVks, IMaciVerifyingKeys } from "./types";
```

Comment:
I think key might be a bit too general, looks like all functions are related to verifying keys here which is more explicit